src/jit/codegenarm.cpp

```diff
@@ -1414,6 +1465,18 @@ instruction CodeGen::genGetInsForOper(genTreeOps oper, var_types type)
         case GT_ROR:
             ins = INS_ror;
             break;
+        case GT_ADD_LO:
+            ins = INS_add;
```

Comment:
Thank you for the feedback, I forgot to introduce setCarry variable here from codegenlegacy.cpp. I will update this.